docs/rowAndColumnTables.md

```diff
@@ -36,7 +36,7 @@ For row format tables column definition can take underlying GemFire XD syntax to
 But for column table it's restricted to Spark syntax for column definition e.g.
 
     snc.sql("CREATE TABLE tableName (Col1 INT ,Col2 INT, Col3 INT) USING column options(BUCKETS '5')" )
-Clauses like PRIMARY KEY , NOT NUL etc. are not supported for column definition. 
+Clauses like PRIMARY KEY, NOT NUL etc. are not supported for column definition. 
```

Comment:
Sure, missed that. I'll take care of it today.

On Thursday, 17 March 2016, hbhanawat <notifications@github.com> wrote:

> In docs/rowAndColumnTables.md
> <https://github.com/SnappyDataInc/snappydata/pull/198#discussion_r56455250>
> :
>
> > @@ -36,7 +36,7 @@ For row format tables column definition can take underlying GemFire XD syntax to
> >  But for column table it's restricted to Spark syntax for column definition e.g.
> >
> >      snc.sql("CREATE TABLE tableName (Col1 INT ,Col2 INT, Col3 INT) USING column options(BUCKETS '5')" )
> > -Clauses like PRIMARY KEY , NOT NUL etc. are not supported for column definition.
> > +Clauses like PRIMARY KEY, NOT NUL etc. are not supported for column definition.
>
> Since you are already doing it, can you fix the NUL to be NULL?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/SnappyDataInc/snappydata/pull/198/files/1c7bcaa0b9cfe1a8653ef63549e743a198ef9b53#r56455250>
>